src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewIconCell.swift

```diff
@@ -38,9 +38,9 @@ class DynamicTableViewIconCell: UITableViewCell {
 
 		NSLayoutConstraint.activate([
 			stackView.leadingAnchor.constraint(equalTo: contentView.layoutMarginsGuide.leadingAnchor),
-			stackView.topAnchor.constraint(equalTo: contentView.layoutMarginsGuide.topAnchor),
+			stackView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 12),
 			stackView.trailingAnchor.constraint(equalTo: contentView.layoutMarginsGuide.trailingAnchor),
-			stackView.bottomAnchor.constraint(equalTo: contentView.layoutMarginsGuide.bottomAnchor),
+			stackView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -12),
```

Comment:
Did you check that this does not break other views?

Reply:
This change fixes a layout glitch while scrolling the table.
I checked some other screens. They looked good.
**I did not check every screen.** 

Several options here:

1. Revert this change and we still have this glitch.
2. Create another component (copy this and change the constraint) to only have the new component at this place. This would create more code to maintain.
3. Keep this change, check the UI-Tests and Screenshot.

I would go with point 3. But I'm also OK with the others.
@marcussc What do you think?

Reply:
3. seems fine to me too, just wanted to have some awareness :)